lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
BTW, I've already changed this back: https://github.com/zverok/daru/commit/9d4d333095d1e7ea98318d8d7ff2ef0ab9570c25

Don't sure why it is not visible in pull request.